internal/handlers/lokistack_create_or_update.go

```diff
@@ -63,11 +63,13 @@ func CreateOrUpdateLokiStack(ctx context.Context, req ctrl.Request, k k8s.Client
 
 	// Here we will translate the lokiv1beta1.LokiStack options into manifest options
 	opts := manifests.Options{
-		Name:          req.Name,
-		Namespace:     req.Namespace,
-		Image:         img,
-		Stack:         stack.Spec,
-		ObjectStorage: *storage,
+		Name:                        req.Name,
+		Namespace:                   req.Namespace,
+		Image:                       img,
+		Stack:                       stack.Spec,
+		ObjectStorage:               *storage,
+		UseServiceMonitors:          manifests.UseServiceMonitors,
+		UseTLSEnabledServiceMonitor: manifests.UseTLSEnabledServiceMonitors,
```

Comment:
nit. Suggest to switch from `Use` to `Enable` as a prefix, as we literally speak of enabling a feature via this flag.